workitem/workitemtype_repository.go

```diff
@@ -230,7 +230,7 @@ func convertFieldTypeToModels(t app.FieldType) (FieldType, error) {
 			return nil, errs.WithStack(err)
 		}
 		if !componentType.isSimpleType() {
-			return nil, fmt.Errorf("Component type is not list type: %s", componentType)
+			return nil, fmt.Errorf("Component type is not list type: %s", reflect.ValueOf(componentType).Type())
```

Comment:
Or you can use `return nil, fmt.Errorf("Component type is not list type: %T", componentType)` instead.